Formula/clang-format.rb

```diff
@@ -33,10 +34,10 @@ class ClangFormat < Formula
 
   depends_on "cmake" => :build
   depends_on "ninja" => :build
+  depends_on "python@3.10"
 
   uses_from_macos "libxml2"
   uses_from_macos "ncurses"
-  uses_from_macos "python", since: :catalina
```

Comment:
It seems like this should be done when the symlinks are migrated, no?

Reply:
If you really want to migrate it before then, you could turn this into an `on_linux` block.